src/components/organisms/AdminVenueView/components/DateTimeField/DateTimeField.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useCallback, useEffect, useState } from "react";
+
+import { generateId } from "utils/id";
+import { getDateTimeFromUtc, getUtcFromDateTime } from "utils/time";
+
+import "./DateTimeField.scss";
+
+export interface DateFieldProps {
+  title: string;
+  subTitle?: string;
+  name: string;
+  dateTimeValue?: number;
+  onChange: (value: number) => void;
+}
+
+export const DateTimeField: React.FC<DateFieldProps> = ({
+  title,
+  subTitle,
+  name,
+  dateTimeValue,
+  onChange,
+}) => {
+  const { date, time } = getDateTimeFromUtc(dateTimeValue);
+
+  const [dateValue, setDateValue] = useState(date);
+  const [timeValue, setTimeValue] = useState(time);
+  const handleDateChange = useCallback((e) => setDateValue(e.target.value), []);
+  const handleTimeChange = useCallback((e) => setTimeValue(e.target.value), []);
+
+  useEffect(() => {
+    const dateTime = getUtcFromDateTime(`${dateValue} ${timeValue}`);
+
+    onChange(dateTime);
+  }, [dateValue, timeValue, onChange]);
+
+  return (
+    <label className="DateTimeField__title" id={generateId(name)}>
```

Comment:
The pairing goes like this
- `<label htmlFor= ...`
- `<input id=...`

but that's only there for cases when `label` doesn't include `input` as child. 

Since this is the case, there is no longer need of any `id` prop.
```suggestion
    <label className="DateTimeField__title">
```

Reply:
From MDN (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#attr-for):

>Note: A <label> element can have both a for attribute and a contained control element, as long as the for attribute points to the contained control element.

Reply:
Unfortunately, per https://stackoverflow.com/a/9004357, it turns out that only a single `input` can be associated to a `label` so it might be better to change it to `div`, and if necessary, wrap both `input` tags with their own `label` tag, or have them connected with 2 separate `id`/`htmlFor` pairs or follow the SO answer for an alternative

Reply:
thank you a lot for detailed explanation. Removed `id` and set Label as a parent for this input

Reply:
fixed in b16a405ab6403c476874cd1cb2bab1f3107abdba